jwt/jwks_client.py

```diff
@@ -20,9 +20,9 @@ def __init__(
         cache_keys: bool = False,
         max_cached_keys: int = 16,
         cache_jwk_set: bool = True,
-        lifespan: int = 300,
+        lifespan: float = 300,
```

Comment:
is 300 a int or float?

Reply:
> Then why using float here?

Maybe to avoid code like this:

```python
KEYCLOAK_CERTS_URL = "..."
KEYCLOAK_TIMEOUT = 0.5
jwks_client = jwt.PyJWKClient(
    KEYCLOAK_CERTS_URL,
    # Error caused by typing bug in `pyjwt`
    timeout=KEYCLOAK_TIMEOUT,  # type: ignore[arg-type]
)
```